sql/plsql/PlSqlParser.g4

```diff
@@ -4004,7 +4004,9 @@ deallocate_unused_clause
     ;
 
 shrink_clause
-    : SHRINK SPACE_KEYWORD COMPACT? CASCADE?
+    // CHECK is an internal Oracle option
+    // It is used to check for proper segment type and segment attributes to allow shrink
+    : SHRINK SPACE_KEYWORD (COMPACT|CASCADE|CHECK)?
```

Comment:
Please use spaces before and after `|` (the grammar is written in such style):

```suggestion
    : SHRINK SPACE_KEYWORD (COMPACT | CASCADE | CHECK)?
```

Reply:
fixed